ui/app/templates/vault/cluster/clients.hbs

```diff
@@ -24,11 +24,13 @@
           Entity/Non-entity clients
         </LinkTo>
       </li>
-      <li>
-        <LinkTo @route="vault.cluster.clients.counts.sync" data-test-tab="sync">
-          Secrets sync clients
-        </LinkTo>
-      </li>
+      {{#if this.hasSecretsSync}}
```

Comment:
purposefully using `this.hasSecretsSync` here (which only checks if you have access to secrets sync, it does NOT check if it has been activated) because we want to show the tab if you have access but just haven't activated it yet

Reply:
My understanding here is that the billing is only of value to users (and will only have data) if the feature is activated, so I would think we'd want to use the same is activated check here 🤔

Reply:
Sorry - following up from slack. I forgot the updated empty state prompted activation so good call to keep this just the feature check!